docs/content/how-to/automated-machine-management/targeted-scale-down.md

```
@@ -0,0 +1,57 @@
+
+You annotate one or more Nodes in the hosted cluster with `hypershift.openshift.io/scale-down=true`. The HCCO Node Controller detects this annotation and sets `cluster.x-k8s.io/delete-machine=yes` on the corresponding Machine in the management cluster. When a scale-down occurs (either by reducing `spec.replicas` on the NodePool or via the cluster autoscaler), CAPI prioritizes machines with the `delete-machine` annotation for deletion.
+
+!!! important
+
+    This annotation is a **deletion-priority hint**, not a trigger. Annotating a Node does **not** cause it to be deleted immediately. The machine is only removed when a scale-down actually occurs.
+
+## Usage
+
+### Marking a node for preferential deletion
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)